### PR TITLE
Prepare environment to run Observability Labs without running previous labs.  

### PR DESCRIPTION
**Pull request by lyakhovv**
#### What this PR does / why we need it:
Added information  to index.md file to use `reset-environment ` command to prepare the environment if the customer did not complete the first setup module.

#### Which issue(s) this PR fixes:
https://github.com/aws-samples/eks-workshop-v2/issues/495 
Fixes #
Added a Note with information to deploy all modules.

_No changes to underline code or commands. _ 

#### Quality checks

- [X ] My content adheres to the style guidelines
- [ ] I ran `make test` or `make e2e-test` and it was successful

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: website/docs/observability/index.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ In this chapter,  we'll cover how you can use AWS observability solutions integr
 * Monitoring Metrics with CloudWatch Container Insights
 * Monitoring EKS Metrics with AMP and ADOT.
 
+:::tip If you did not a complete the [Introduction Getting Started session](../introduction/getting-started/finish.md) you can
```
Comment (niallthomson) [inline website/docs/observability/index.md:16]: This is not necessary since it isn't a lab

File: website/docs/observability/logging/index.md [hunk 2]
```diff
@@ -10,3 +10,13 @@ Containers are grouped as Pods within a Kubernetes cluster and are scheduled to
 In this lab, we'll see 
  - How to enable EKS Control Plane logs and verify it in the Amazon CloudWatch
  - How to setup the logging agent (Fluent Bit) to stream the Pod logs to Amazon CloudWatch
+
+
+:::tip Before you start
```
Comment (niallthomson) [inline website/docs/observability/logging/index.md:15]: This is not necessary since it isn't a lab

**Review by niallthomson — CHANGES_REQUESTED** [review]
Hey @lyakhovv looks fine except for the comments I left.

File: website/docs/observability/logging/index.md (hunk 2 above)
Comment (lyakhovv) [reply website/docs/observability/logging/index.md]: Will remove the note

File: website/docs/observability/index.md (hunk 1 above)
Comment (lyakhovv) [reply website/docs/observability/index.md]: Done

**Comment by lyakhovv** [comment]
Please let me know if there are any additional blockers for this MR

**Comment by niallthomson** [comment]
Thanks @lyakhovv !

**Pull request merged**